spec/MongoTransform.spec.js

```diff
@@ -320,6 +320,9 @@ describe('parseObjectToMongoObjectForCreate', () => {
     done();
   });
 
+});
+
+describe_only_db('mongo')('parseObjectToMongoObjectForCreate', () => {
```

Comment:
I was writing you to ask if this was what you where asking for. I don't know how to make this tests. Is there any edge to edge test case in the repository? Sorry, I'm not really familiar with Jasmine.

If you tell me how to make this tests or where to take an idea, I'll take a look for sure.

Do you think I should rollback this change??